src/main/kotlin/no/skatteetaten/aurora/gobo/ApplicationConfig.kt

```diff
@@ -58,6 +58,7 @@ private val logger = KotlinLogging.logger {}
 
 @Configuration
 class ApplicationConfig(
+    val webClientBuilder: WebClient.Builder,
```

Comment:
Her har vel Jarle også gjort noen endringer i en annen gobo sak så man må merge dette sammen egentlig.

Reply:
Tror ikke den er merget til master enda. Skal høre med Jarle for å se om merger den inn nå.